### ISSUE TITLE
poi-docker #33 failed

### ISSUE THREAD

**Issue by vineet96** [vineet96/openhack-devops-team#30]
Build 'poi-docker' is failing!

Last 50 lines of build output:

```
[...truncated 150 lines...]
WARNING! Your password will be stored unencrypted in /var/lib/jenkins/workspace/poi-docker/apis/poi/web@tmp/3df18d43-1f83-4b8f-a6b0-174e9b29ed91/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded
[Pipeline] {
[Pipeline] sh
+ docker push openhackipi0acr.azurecr.io/devopsoh/api-poi:33
The push refers to repository [openhackipi0acr.azurecr.io/devopsoh/api-poi]
6f43ccd3630f: Preparing
70e288ca0288: Preparing
b29986f25fdb: Preparing
b116468880ac: Preparing
57bda236ae67: Preparing
ef68f6734aa4: Preparing
ef68f6734aa4: Waiting
70e288ca0288: Layer already exists
6f43ccd3630f: Layer already exists
b116468880ac: Layer already exists
b29986f25fdb: Layer already exists
ef68f6734aa4: Layer already exists
57bda236ae67: Layer already exists
33: digest: sha256:f55168bc20c9a0b013fa67049b0de93a5e86c333dedc8e07d76ffdc915d429bd size: 1581
[Pipeline] }
[Pipeline] // withDockerRegistry
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Deploy to AKS)
[Pipeline] dir
Running in /var/lib/jenkins/workspace/poi-docker/apis/poi
[Pipeline] {
[Pipeline] sh
+ helm upgrade --install --force --values helm/values.yaml --set repository.image=openhackipi0acr.azurecr.io/devopsoh/api-poi,repository.tag=33,ingress.rules.endpoint.host=akstraefikopenhackipi0.eastus.cloudapp.azure.com --wait api-poi helm
Error: UPGRADE FAILED: a released named api-poi is in use, cannot re-use a name that is still in use
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] script
[Pipeline] {
[Pipeline] properties
[Pipeline] }
[Pipeline] // script
[Pipeline] step

```

Changes since last successful build:
- [TSwift] f0d5b32f625d129969442a19dbd76ebc87733d34 - update deployment.yaml for bluegreen

[View full output](http://localhost:8080/job/poi-docker/33/)

**Comment by vineet96** [comment]
Build 'poi-docker' is failing!

Last 50 lines of build output:

```
[...truncated 150 lines...]
WARNING! Your password will be stored unencrypted in /var/lib/jenkins/workspace/poi-docker/apis/poi/web@tmp/895bd2b4-3235-4a0c-9e84-b3ec378efbbe/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded
[Pipeline] {
[Pipeline] sh
+ docker push openhackipi0acr.azurecr.io/devopsoh/api-poi:34
The push refers to repository [openhackipi0acr.azurecr.io/devopsoh/api-poi]
6f43ccd3630f: Preparing
70e288ca0288: Preparing
b29986f25fdb: Preparing
b116468880ac: Preparing
57bda236ae67: Preparing
ef68f6734aa4: Preparing
ef68f6734aa4: Waiting
b29986f25fdb: Layer already exists
b116468880ac: Layer already exists
57bda236ae67: Layer already exists
6f43ccd3630f: Layer already exists
ef68f6734aa4: Layer already exists
70e288ca0288: Layer already exists
34: digest: sha256:f55168bc20c9a0b013fa67049b0de93a5e86c333dedc8e07d76ffdc915d429bd size: 1581
[Pipeline] }
[Pipeline] // withDockerRegistry
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Deploy to AKS)
[Pipeline] dir
Running in /var/lib/jenkins/workspace/poi-docker/apis/poi
[Pipeline] {
[Pipeline] sh
+ helm upgrade --install --force --values helm/values.yaml --set repository.image=openhackipi0acr.azurecr.io/devopsoh/api-poi,repository.tag=34,ingress.rules.endpoint.host=akstraefikopenhackipi0.eastus.cloudapp.azure.com --wait api-poi helm
Error: UPGRADE FAILED: a released named api-poi is in use, cannot re-use a name that is still in use
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] script
[Pipeline] {
[Pipeline] properties
[Pipeline] }
[Pipeline] // script
[Pipeline] step

```

Changes since last successful build:
- [TSwift] f0d5b32f625d129969442a19dbd76ebc87733d34 - update deployment.yaml for bluegreen

- [aleksandr.castillo] 5343764c3e1f9b4000e0f341018f550f8d9b269e - fix yaml (#31)



[View full output](http://localhost:8080/job/poi-docker/34/)

**Comment by vineet96** [comment]
Build was fixed!
